open_bounties/0062 Account-based toml Lookup

```diff
@@ -0,0 +1,40 @@
+---
+ID: "0062"
+Discussion: https://github.com/XRPLBounties/Proposals/discussions/0062
+Category: XRPL.org tools
+Status: Open
+Amount: $2,500
+---
+
+
+# Proposal Title
+
+Account-based toml Lookup Tools
+
+**Category:**
+XRPL.org tools
+
+
+**Amount:** 
+$2,500
+
+## Overview
+
+The xrp-ledger.toml checker on XRPL.org (https://xrpl.org/xrp-ledger-toml-checker.html) is a handy tool for looking up what accounts are claimed by a specific domain (including domain verification), but it could be convenient to be able to do it the other way around and check to see if an account really is claimed by the domain it has set in its Domain field.
+
+So in addition to being able to put in a domain and verify the toml from there, you could put in an account to verify that account's domain: if it has a Domain field, you look up to see if that domain serves a toml file and if that toml claims ownership of the account.
+
+Both milestones must be completed (reviewed and approved) together in a PR to claim the bounty. 
```

Comment:
I think if they need to be completed together, they should be one milestone. You can write out the detailed requirements in this Overview section, then refer back to them in the milestone description.

Reply:
moved a few things around, let me know what you think